cfgov/v1/management/commands/makemessages.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# This command overrides the built-in Django makemessages command to extract
+# messages from both Django templates and Jinja2 templates.
+#
+# The code is based on this StackOverflow question and answer:
+# https://stackoverflow.com/questions/2090717/getting-translation-strings-for-jinja2-templates-integrated-with-django-1-x
+# And the Django docs:
+# https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#customizing-the-makemessages-command
+
+import re
+
+from django import VERSION
+from django.core.management.commands import makemessages
+
+if VERSION[:2] < (1, 11):
+    from django.utils.translation import trans_real
+else:
+    from django.utils.translation import template as trans_real
+
+
+class Command(makemessages.Command):
+
+    def handle(self, *args, **options):
+        # Jinja2 uses {% trans %}…{% endtrans %}
+        jinja_block_re = re.compile(r'^\s*trans(?:\s+|$)')
+        jinja_endblock_re = re.compile(r'^\s*endtrans$')
```

Comment:
Should this be updated to support your addition of the standard `gettext` and `_` methods as things that can be called in Jinja?

Reply:
Django's `trans_real` already matches [`_()`](https://github.com/django/django/blob/stable/1.8.x/django/utils/translation/trans_real.py#L548). I could add `gettext()`, but I think it'd be better to stick to the `_()` convention. This also doesn't prevent just adding the string to the `.po` file manually. `makemessages` is purely for automating that step.

Reply:
I [added an additional comment along these lines below](https://github.com/cfpb/cfgov-refresh/pull/3295/files#diff-15ca6490a115a0eeaffbf9774721a606R32).